infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -274,23 +275,24 @@ private void initializationTask() {
             }
 
             // Initiating periodic task to poll for failures.
-            try {
-                localMetricsPollingService.scheduleAtFixedRate(
-                        this::updateLocalMetrics,
```

Comment:
break up the runRecoveryReconfiguration so that the sequencer and epoch changes are separate.